addons/elasticsearch-data/7.8.x/elasticsearch-data-1.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: elasticsearch
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: elasticsearch-data
```

Comment:
Also you're adding 3 new elasticsearch addons? Shouldn't we group all of them as subcharts? We already have a customer issue of having so many different elasticsearch addons that are actually dependent of each other and customers don't know what they can enable/disable.